### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by basitanwer**
Pointing this change to tallat
I'll give him the link.

### CODE REVIEW

**Review by amin3141 — APPROVED** [review]
I'm assuming you tested this? Approved, you can go ahead and merge it.

File: start.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+rasa run actions & rasa run --credentials ./credentials.yml  --enable-api --auth-token XYZ123 --model ./models --endpoints ./endpoints.yml --cors "*"
```
Comment (tallat-nuage) [inline start.sh]: Where will the log for "rasa run actions" go? Perhaps redirect to terminal too, or to some file in /tmp?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+FROM rasa/rasa:2.7.1-spacy-en AS trained
+
+WORKDIR /hotel-bot
+COPY . /hotel-bot
+
+RUN rasa train
+
+FROM rasa/rasa:2.7.1-spacy-en
```
Comment (tallat-nuage) [inline Dockerfile:10]: Any reason why you have broken this down into two separate steps? As in, why use an intermediate image?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,16 @@
+FROM rasa/rasa:2.7.1-spacy-en AS trained
+
+WORKDIR /hotel-bot
```
Comment (tallat-nuage) [inline Dockerfile]: Lets use /opt/zir/hotel-bot

File: start.sh (hunk 1 above)
Comment (basitanwer) [reply start.sh]: Fixed the logging

**Comment by basitanwer** [comment]
@tallat-nuage Please review the changes again.

**Pull request merged**